.github/workflows/scorecards-analysis.yml

```diff
@@ -0,0 +1,55 @@
+name: Scorecards supply-chain security
+on:
+  # Only the default branch is supported.
+  branch_protection_rule:
+  schedule:
+    - cron: '39 19 * * 6'
```

Comment:
Any chance of a comment here to explain what this cron spec is for us mortals?

https://crontab.guru/#39_19_*_*_6 seems to think this means "At 19:39 on Saturday" which is... specific? Why this time and day?

Reply:
> Why this time and day?

@JamesLaverack -- No reason outside of these being the default cron values in the CodeQL + Scorecard actions! We can tweak them later, as needed.

Reply:
Fair enough.
/lgtm